src/handlers/postData.js

```diff
@@ -0,0 +1,10 @@
+const db_connection = require('../../database/db_connection');
+
+const postData = (userDetails, cb) => {
+  db_connection.query('INSERT INTO users (name, location) VALUES ($1, $2)', [userDetails.name, userDetails.location], (err, res) => {
+    if (err) {
+      return cb(err);
```

Comment:
yes this one is fine. but I think confusing for beginners (since the return is being used to exit the function, but not to return anything)